openedx/core/djangoapps/profile_images/tests/test_views.py

```diff
@@ -219,6 +221,57 @@ def test_upload_self(self, mock_make_image_version, mock_log):  # pylint: disabl
 
         self.check_upload_event_emitted(old=TEST_UPLOAD_DT, new=TEST_UPLOAD_DT2)
 
+    @ddt.data(
+        ('image/jpeg', '.jpg'),
+        ('image/jpeg', '.jpeg'),
+        ('image/pjpeg', '.jpg'),
+        ('image/pjpeg', '.jpeg'),
+        ('image/png', '.png'),
+        ('image/gif', '.gif'),
+        ('image/gif', '.GIF'),
```

Comment:
I could put a ddt on test_upload_self...
On Oct 8, 2015 4:14 PM, "Daniel Friedman" <notifications@github.com> wrote:

> In openedx/core/djangoapps/profile_images/tests/test_views.py
> <https://github.com/edx/edx-platform/pull/10046#discussion_r41562733>:
>
> > @@ -219,6 +221,57 @@ def test_upload_self(self, mock_make_image_version, mock_log):  # pylint: disabl
> >
> >          self.check_upload_event_emitted(old=TEST_UPLOAD_DT, new=TEST_UPLOAD_DT2)
> >
> > +    @ddt.data(
> > +        ('image/jpeg', '.jpg'),
> > +        ('image/jpeg', '.jpeg'),
> > +        ('image/pjpeg', '.jpg'),
> > +        ('image/pjpeg', '.jpeg'),
> > +        ('image/png', '.png'),
> > +        ('image/gif', '.gif'),
> > +        ('image/gif', '.GIF'),
>
> Though it would be testing existing functionality, I like the idea of
> extending this ddt data to also cover content-type=multipart/form-data,
> since there's no other test that I see here which tests all the mimetypes
> with that upload method.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10046/files#r41562733>.
>